src/blockchain/log-processors/designated-report-submitted.ts

```diff
@@ -3,7 +3,35 @@ import * as Knex from "knex";
 import { FormattedLog, ErrorCallback } from "../../types";
 
 export function processDesignatedReportSubmittedLog(db: Knex, augur: Augur, trx: Knex.Transaction, log: FormattedLog, callback: ErrorCallback): void {
-    console.log("TODO: DesignatedReportSubmitted");
-    console.log(log);
-    callback(null);
+    // @achapman: "The designated reporter purchases Stake Tokens in order to do the Designated Report"
+    // Should designatedReportSubmitted include StakedToken address?
+
+    // for phase, sqlite lacks an ENUM, but the idea here is that:
+    //  phase:null = no report submitted
+    //  phase:0 = designated report submitted
+    //  phase:1 = round1
+    //  phase:2 = round2
+    // if that seems reasonable, i'll update PR to import the enum and use name below
+    const marketStateDataToInsert: { [index: string]: string | number | boolean } = {
```

Comment:
Minor style point: elsewhere we've not put spaces around the `|` in types, i.e., `string|number` instead of `string | number`.  I don't have a strong preference for which style we use but I think it should be consistent throughout the code!

Reply:
(This occurs elsewhere too, but only explicitly mentioning it here.)

Reply:
VSCode adds those spaces when you do the auto-format, i wonder if i can change that.

Reply:
Hm, if you can't easily it's not a big deal, just style.